src/cargo.rs

```diff
@@ -149,7 +152,7 @@ pub struct Config {
 
 impl Config {
     pub fn target(&self) -> Result<Option<String>> {
-        if let Some(v) = self.table.lookup("build.target") {
+        if let Some(v) = self.table.get("build.target") {
```

Comment:
The `get` method does not support the `.` separator like the `lookup` method did. So you instead have to do something like this:

```suggestion
        if let Some(v) = self.table.get("build").and_then(|v| v.get("target")) {
```

This is also applies to the other `lookup` → `get` changes.

Reply:
Whoops! I didn't even realize that. Thanks for the sharp eyes.